assets/src/edit-story/components/library/panes/media/mediaEditDialog.js

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import { useCallback, useEffect, useState } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useAPI } from '../../../../app/api';
+import Dialog from '../../../../components/dialog';
+import { Plain } from '../../../../components/button';
+
+const styledMediaThumbnail = css`
+  display: flex;
+  width: 152px;
+  margin-right: 28px;
+`;
+
+const Image = styled.img`
+  ${styledMediaThumbnail}
+`;
+
+const Video = styled.video`
+  ${styledMediaThumbnail}
+`;
+
+const DialogBody = styled.div`
+  max-width: 452px;
+  display: flex;
+  justify-content: left;
+  align-items: flex-start;
+`;
+
+const MetadataTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const MediaTitleText = styled.div`
+  font-weight: bold;
+  color: #3c4043;
+`;
+
+const MediaSizeText = styled.div`
+  font-size: 16px;
+  color: #3c4043;
+`;
+
+const Input = styled.input`
+  background: #ffffff;
+  border: 1px solid #d3d4d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+  font-size: 15px;
+  padding: 7px 10px;
+  margin-top: 20px;
+  margin-bottom: 4px;
+`;
+
+const DialogDescription = styled.div`
+  font-size: 12px;
+  line-height: 16px;
+  color: #202124;
+  opacity: 0.54;
```

Comment:
Should we use something like this 
https://github.com/google/web-stories-wp/blob/08952b335aff7dc906ad6edb5141e916b574cd78/assets/src/edit-story/theme.js#L125